service/src/main/kotlin/vdi/service/InstallMetaHandler.kt

```diff
@@ -58,4 +59,11 @@ class InstallMetaHandler(
 
     timer.observeDuration()
   }
+
```

Comment:
Does this need to be duplicated?  I guess you're thinking in the future we may add a value to one of the envs but not the other and they just happen to be the same right now?

Reply:
Basically, yeah.  It's just coincidence that they both need it, and they are the only 2 out of the 4 total.